test/compilationTests/gnosis/MarketMakers/LMSRMarketMaker.sol

```diff
@@ -128,7 +128,9 @@ contract LMSRMarketMaker is MarketMaker {
     /// @param netOutcomeTokensSold Net outcome tokens sold by market
     /// @param funding Initial funding for market
     /// @param outcomeIndex Index of exponential term to extract (for use by marginal price function)
-    /// @return A result structure composed of the sum, the offset used, and the summand associated with the supplied index
+    /// @return sum of the outcomes
```

Comment:
Ok, that's probably nit-picking, but these could be changed to ``/// @return sum The Sum of outcomes`` etc.